pkg/exporter/process.go

```diff
@@ -84,7 +89,9 @@ type ExporterInput struct {
 func InitExportingProcess(input ExporterInput) (*ExportingProcess, error) {
 	var conn net.Conn
 	var err error
-
+	if input.CollectorProtocol == "udp" && input.SendJSONRecord {
```

Comment:
Hi Yiou, I thought about this more. Even though UDP packet length is large here.. IP fragmentation happens and packet get reassembled in Logstash instance. There is possibility of record loss, but that is ok. Logstash received records fine with IP fragmentation. I checked with large records and UDP transport. 
You can remove this check.

Reply:
Got it. Updated. will add some local tests later.

Reply:
I did not mean local unit tests. I just meant a test on local setup with Antrea+local ELK flow collector.